plugins/pangea/manifest.json

```diff
@@ -0,0 +1,75 @@
+{
+  "id": "pangea",
+  "description": "Pangea Text Guard for scanning LLM inputs and outputs",
+  "credentials": {
+    "type": "object",
+    "properties": {
+      "token": {
+        "type": "string",
+        "label": "Pangea token",
+        "description": "AI Guard token. Get your token configured on Pangea User Console (https://pangea.cloud/docs/getting-started/configure-services/#configure-a-pangea-service).",
+        "encrypted": true
+      },
```

Comment:
Can we rename it to `apiKey` please, since that's the naming we're using for other plugins which will make it easier for others to easily switch to new plugin?

Reply:
Renamed!